src/event/handlers/CodeblocksOverFileUploadsHandler.ts

```diff
@@ -0,0 +1,35 @@
+import { Constants, MessageEmbed, Message } from "discord.js";
+import EventHandler from "../../abstracts/EventHandler";
+import { allowed_file_extensions } from "../../config.json";
+
+class CodeblocksOverFileUploadsHandler extends EventHandler {
+	constructor() {
+		super(Constants.Events.MESSAGE_CREATE);
+	}
+
+	async handle(message: Message): Promise<void> {
+		let invalidFileFlag = false;
+
+		if (message.attachments.size > 0)		{
```

Comment:
Can you change the massive cap between the `(` and `{` to a single space, please?